README.md

```diff
@@ -7,11 +7,116 @@ The SDK is compatible with:
 * .NET Framework 4.6.1
 * Mono
 
-## Development version (1.x)
+## Installation
+### Using Nuget Package Manager
+ 1. Right-click on your project in the Solution Explorer and choose **Manage Nuget Packages...**
+ 2. Search for Okta. Install the `Okta.Sdk` package.
 
-The [master](https://github.com/okta/okta-sdk-dotnet/tree/master) branch contains the 1.x version of the SDK, which is currently under active development. This version is a major update, and is a breaking change from all previously-released packages. It has not yet been published to NuGet.
+### Using The Package Manager Console
+Simply run `install-package Okta.Sdk`. Done!
 
-If you want to be notified when updates are released, use the Watch button above to watch this repository!
+## Getting Started
+To use the SDK, you will you an `OktaClient`. The `OktaClient` needs an OrgUrl and an API Token. You can see how to create them [here](https://developer.okta.com/docs/api/getting_started/getting_a_token.html).
+
+## Client Configuration
+
+You can configure the `OktaClient` in one of three ways:
+
+With a `okta.yaml` file either:
+
+* in the root of the project
+* in a .okta folder in the current user's home folder (`~/.okta/okta.yml` on *nix machines, `%userprofile%\.okta\okta.yml` on Windows)
```

Comment:
Minor issue: github thinks that everything after `*nix` is italic because of the asterisk. Can you try `\*nix`?